src/+FeatureCreators/FeatureSet5Blockmean.m

```diff
@@ -0,0 +1,157 @@
+classdef FeatureSet5Blockmean < FeatureCreators.Base
+% FeatureSet3Blockmean Specifies a feature set consisting of:
+%   see FeatureSet3Blockmean.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = FeatureSet5Blockmean( )
+            obj = obj@FeatureCreators.Base();
+            obj.deltasLevels = 2;
+        end
+        %% ----------------------------------------------------------------
+
+        function afeRequests = getAFErequests( obj )
+            commonParams = FeatureCreators.LCDFeatureSet.getCommonAFEParams();
+            afeRequests{1}.name = 'amsFeatures';
+            afeRequests{1}.params = genParStruct( ...

```

Comment:
I'm unsure what's better with respect to speed. AMS/16, resulting in two complete sets of filtering (16 and 32) all stages, or AMS/32, having only one set of Gammatone filters, but on the other hand 32x8 mod filters instead of 16x8...

This is probably primarily a question to @Hardcorehobel . What do you think?